tests/utils/test_embeddings.py

```diff
@@ -314,6 +326,41 @@ def testRasterEmbeddingSearchBasic(self):
                 len(set(value_list)), len(value_list), "embeddings overlap"
             )
 
+    def testFindSublatticeEmbeddingsTile(self):
+        # Check function responds correctly to tile specification
+        topology = "chimera"
+        min_graph_rows = 1
+        S = nx.from_edgelist({(i, i + 1) for i in range(5)})  # 6 nodes
+        T = dnx.chimera_graph(min_graph_rows + 1)
+        tile = dnx.chimera_graph(min_graph_rows, node_list=list(range(1, 8)))
+        embs = find_sublattice_embeddings(
+            S, T, graph_rows=min_graph_rows, max_num_emb=float("Inf"), tile=tile
+        )
+        self.assertEqual(len(embs), 4)
+        nodes_used = {v for emb in embs for v in emb.values()}
+        self.assertEqual(len(nodes_used), S.number_of_nodes() * len(embs))
+        self.assertTrue(
+            all(n % 8 > 0 for n in nodes_used), "Every 8th node excluded by tile"
+        )
+        tile5 = dnx.chimera_graph(min_graph_rows, node_list=list(range(3, 8)))
+        embs = find_sublattice_embeddings(
+            S, T, graph_rows=min_graph_rows, max_num_emb=float("Inf"), tile=tile5
+        )
+        self.assertEqual(len(embs), 0, "Tile is too small")
+
+        S = tile
```

Comment:
break into multiple tests

Reply:
TestTile1 and TestTile2? Some of the same objects are being used throughout, and its only a test of one input argument, therefore I was thinking it was appropriate to keep all the tile variation tests together.

Reply:
Wouldn't it make sense to keep all the tests associated to the input parameter 'tile' together, especially since the tests share input object and context.